en/docs/learn/api-security/oauth2/oauth2-scopes/fine-grained-access-control-with-oauth-scopes.md

```diff
@@ -4,95 +4,216 @@ Scopes enable fine-grained access control to API resources based on user roles.
+Fill in required details of the scope and click **Save** button.
 
     <table>
-    <tbody>
-    <tr class="odd">
-    <td><strong>Scope Key</strong></td>
-    <td>A unique key for identifying the scope. Typically, it is prefixed by part of the API's name for uniqueness, but is not necessarily reader-friendly.</td>
-    </tr>
-    <tr class="even">
-    <td><strong>Scope Name</strong></td>
-    <td>A human-readable name for the scope. It typically says what the scope does.</td>
-    </tr>
-    <tr class="odd">
-    <td><strong>Roles</strong></td>
-    <td><div class="content-wrapper">
-    <p>The user role(s) that are allowed to obtain a token against this scope. E.g., manager, employee.</p>
-    <p>Note that the role name is <strong>case sensitive</strong> in the DBMSs that are case sensitive, such as PostgreSQL.</p>
-    <p>When the role you specify is in a secondary user store, you have to give the role as <code>&lt;userstore name&gt;/&lt;role name&gt;</code>.</p>
-    
-    </div></td>
-    </tr>
-    </tbody>
+     <tbody>
+          <tr class="odd">
+               <td><strong>Scope Name</strong></td>
+               <td>A unique key for identifying the scope. This should be unique across all the APIs of the tenant.</td>
+          </tr>
+          <tr class="even">
+               <td><strong>Scope Description</strong></td>
+               <td>The description for the scope.</td>
+          </tr>          
+          <tr class="odd">
+               <td><strong>Roles</strong></td>
+               <td><div class="content-wrapper">
+                    <p>The user role(s) that are allowed to obtain a token against this scope. E.g., manager, employee.</p>
+                    <p>Note that the role name is <strong>case sensitive</strong> in the DBMSs that are case sensitive, such as PostgreSQL.</p>
+                    <p>When the role you specify is in a secondary user store, you have to give the role as <code>&lt;userstore name&gt;/&lt;role name&gt;</code>.</p>
+              </div></td>
+          </tr>
+     </tbody>
     </table>
 
-    <a href="../../../../../assets/img/learn/create-a-new-scope.png" ><img src="../../../../../assets/img/learn/create-a-new-scope.png" alt="Create a Scope" 
+    <a href="{{base_path}}/assets/img/learn/api-security/oauth2/oauth2-scopes/api-scopes/create-a-new-scope.png" >
+    <img src="{{base_path}}/assets/img/learn/api-security/oauth2/oauth2-scopes/api-scopes/create-a-new-scope.png" alt="Create a Scope" 
          title="Create a Scope" width="70%" /></a>
 
-## Applying a Scope to an API Resource
+### Applying a Local Scope to a Resource
 
-A scopes has be applied to a resource, in order to restrict the access to a user group/groups. Please follow below steps to apply a scope to a resource.
+A Local scope has be applied to a resource, in order to restrict the access to a user group/groups. A Local Scope can only be
```

Comment:
Shall we rephrase it to : "A local scope, when applied to a resource, will restrict access for a user group(s)."

Reply:
done